stripe/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt

```diff
@@ -183,16 +183,16 @@ data class PaymentMethodCreateParams internal constructor(
 
     internal enum class Type(internal val code: String, val hasMandate: Boolean = false) {
         Card("card"),
-        Ideal("ideal"),
+        Ideal("ideal", hasMandate = true),
         Fpx("fpx"),
         SepaDebit("sepa_debit", true),
         AuBecsDebit("au_becs_debit", true),
         BacsDebit("bacs_debit", true),
```

Comment:
Should these also say `, hasMandate = true` so it's consistent?

Reply:
@mshafrir-stripe DId you see this one?